src/plugins/gui/providers/bityProvider.ts

```diff
@@ -57,6 +58,12 @@ const CURRENCY_PLUGINID_MAP: StringMap = {
   LTC: 'litecoin'
 }
 
+const EDGE_CLIENT_ID = '4949bf59-c23c-4d71-949e-f5fd56ff815b'
+
+const asBityApiKeys = asObject({
+  clientId: asMaybe(asString)
```

Comment:
change to 

`clientId: asMaybe(asString, EDGE_CLIENT_ID)`

Then remove `EDGE_CLIENT_ID` from line 187